### ISSUE TITLE
RFC 9460 (SVCB / HTTPS) support

### ISSUE THREAD

**Issue by jailbird777** [gdnsd/gdnsd#242]
It would be nice if gdnsd had native support for the SVCB & HTTPS record types

https://datatracker.ietf.org/doc/rfc9460/

**Comment by blblack** [comment]
Agreed, it would be!  They'll most likely be added in the 4.x release series though (don't ask me when that's happening!), as adding new types is much simpler there.
